### ISSUE TITLE
local vs global attention in further MLM pre-training.

### ISSUE THREAD

**Issue by chrisvdwerf** [allenai/longformer#180]
I am currently looking into the [conversion script](https://colab.research.google.com/github/allenai/longformer/blob/master/scripts/convert_model_to_long.ipynb) to create a longformer from an already pre-trained Roberta implementation. I noticed that the script further pre-trains the Longformer on MLM with only local attention. 

- What exactly is the motive behind not having tokens (e.g. bos and eos) attend globally as well? 
- Would it make sense to alter this if the goal you have in mind is focussed on tasks that require embeddings of the entire sequence? (e.g. unsupervised document similarity)

**Comment by JasonChan7** [comment]
I think the tokenizer used in Roberta caused it. There is no global attention mask in the outputs of Bert/Roberta tokenizer.

**Comment by jojonki** [comment]
I have also the same question. It's not clear how they use global attention during Longformer pre-training.

**Comment by giwhite** [comment]
> I have also the same question. It's not clear how they use global attention during Longformer pre-training.

so do i when i read a paper about MDS, i find it may not use global attention.
